src/XCCDF_POLICY/xccdf_policy_remediate.c

```diff
@@ -1286,7 +1395,204 @@ static int _xccdf_policy_generate_fix_other(struct oscap_list *rules_to_fix, str
+	while (oscap_iterator_has_more(post_it)) {
+		char *command = (char *) oscap_iterator_next(post_it);
+		_write_text_to_fd(output_fd, command);
+		_write_text_to_fd(output_fd, "\n");
+	}
+	oscap_iterator_free(post_it);
+	_write_text_to_fd(output_fd, "%end\n");
+	_write_text_to_fd(output_fd, "\n");
+	return 0;
+}
+
+const char *common_kickstart_header = (
+"# Specify installation method to use for installation\n"
+"# To use a different one comment out the 'url' one below, update\n"
+"# the selected choice with proper options & un-comment it\n"
+"#\n"
+"# Install from an installation tree on a remote server via FTP or HTTP:\n"
+"# --url		the URL to install from\n"
+"#\n"
+"# Example:\n"
+"#\n"
+"# url --url=http://192.168.122.1/image\n"
+"#\n"
+"# Modify concrete URL in the above example appropriately to reflect the actual\n"
+"# environment machine is to be installed in\n"
+"#\n"
+"# Other possible / supported installation methods:\n"
+"# * install from the first CD-ROM/DVD drive on the system:\n"
+"#\n"
+"# cdrom\n"
+"#\n"
+"# * install from a directory of ISO images on a local drive:\n"
+"#\n"
+"# harddrive --partition=hdb2 --dir=/tmp/install-tree\n"
+"#\n"
+"# * install from provided NFS server:\n"
+"#\n"
+"# nfs --server=<hostname> --dir=<directory> [--opts=<nfs options>]\n"
+"#\n"
+"\n"
+"# Set language to use during installation and the default language to use on the installed system (required)\n"
+"lang en_US.UTF-8\n"
+"\n"
+"# Set system keyboard type / layout (required)\n"
+"keyboard --vckeymap us\n"
+"\n"
+"# Configure network information for target system and activate network devices in the installer environment (optional)\n"
+"# --onboot	enable device at a boot time\n"
+"# --device	device to be activated and / or configured with the network command\n"
+"# --bootproto	method to obtain networking configuration for device (default dhcp)\n"
+"# --noipv6	disable IPv6 on this device\n"
+"network --onboot yes --device eth0 --bootproto dhcp --noipv6\n"
+"\n"
+"# Set the system's root password (required)\n"
+"# Plaintext password is: server\n"
+"# Refer to e.g. https://pykickstart.readthedocs.io/en/latest/commands.html#rootpw to see how to create\n"
+"# encrypted password form for different plaintext password\n"
+"rootpw --iscrypted $6$/0RYeeRdK70ynvYz$jH2ZN/80HM6DjndHMxfUF9KIibwipitvizzXDH1zW.fTjyD3RD3tkNdNUaND18B/XqfAUW3vy1uebkBybCuIm0\n"
+"\n"
+"# The selected profile will restrict root login\n"
+"# Add a user that can login and escalate privileges\n"
+"# Plaintext password is: admin123\n"
+"user --name=admin --groups=wheel --password=$6$Ga6ZnIlytrWpuCzO$q0LqT1USHpahzUafQM9jyHCY9BiE5/ahXLNWUMiVQnFGblu0WWGZ1e6icTaCGO4GNgZNtspp1Let/qpM7FMVB0 --iscrypted\n"
+"\n"
+"# Configure firewall settings for the system (optional)\n"
+"# --enabled	reject incoming connections that are not in response to outbound requests\n"
+"# --ssh		allow sshd service through the firewall\n"
+"firewall --enabled --ssh\n"
```

Comment:
I'm pretty sure we can remove this - our testing runs without the `firewall` command as Anaconda by default installs `firewalld`, and installing the `openssh-server` package adds port 22 to it seamlessly.